usb/usb_host_hid/idf_component.yml

```diff
@@ -1,3 +1,4 @@
+## IDF Component Manager Manifest File
 version: "1.0.1"

```

Comment:
In order for this change to be uploaded to ESP registry, we must bump the `version`.
In this case we can only bump the revision number to `1.0.1~1`. Which means: no changes in code; only a metadata change

Reply:
Oh...ok. 
Do we want this line in manifest?